File: smoke/scripts/Smoke_Event_CvFail/flow.c
/*-----------------------------------------------------------------------------
    Name: flow
    Recorded By: automation
    Date of recording: 03/12/2014 02:51:06
    Flow details:
    Modification History:
-----------------------------------------------------------------------------*/

#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include "ns_string.h"

void flow()
{
    ns_web_url ("index",
        "URL=http://127.0.0.1/tours/index.html",
    );

    ns_page_think_time(5.0);
}

File: performance/scripts/JTSScript40K/flow.java
/*-----------------------------------------------------------------------------
    Name: flow
    Recorded By: netstorm
    Date of recording: 01/14/2014 09:51:58
    Flow details:
    Modification History:
-----------------------------------------------------------------------------*/

package com.cavisson.scripts.JTSScript40K;
import pacJnvmApi.NSApi;
public class flow implements NsFlow
{
    public int execute(NSApi nsApi) throws Exception
    {
        int status;
       
		//%Url
        status = nsApi.ns_web_url ("index_html",
            "URL=http://127.0.0.1/file_set/dir00000/class2_3.html",
        );

        return status;
    }
}
